README.md

```diff
@@ -17,10 +17,10 @@ This module provides settings:
 ```hcl
 
 module "nat_gateway_traffic" {
-  source = "../../terraform/terraform-nat-gateway-traffic-alert"
+  source = "hazelops/nat-gateway-traffic-alert/aws"
```

Comment:
@kobrikx what about to specify version of source?

Reply:
add  `version = "~> 1.0" `